doc/vmc.md

```diff
@@ -0,0 +1,226 @@
+* [Genotype.proto](#Genotype.proto)
+ * [Genotype](#org.ga4gh.vmc.Genotype)
+* [Haplotype.proto](#Haplotype.proto)
+ * [Haplotype](#org.ga4gh.vmc.Haplotype)
+* [Interval.proto](#Interval.proto)
+ * [Interval](#org.ga4gh.vmc.Interval)
+* [IntervalEdit.proto](#IntervalEdit.proto)
+ * [IntervalEdit](#org.ga4gh.vmc.IntervalEdit)
+* [SequenceReference.proto](#SequenceReference.proto)
+ * [SequenceReference](#org.ga4gh.vmc.SequenceReference)
+ * [SequenceReference.Namespace](#org.ga4gh.vmc.SequenceReference.Namespace)
+* [Scalar Value Types](#scalar-value-types)
+
+<a name="Allele.proto"/>
+<p align="right"><a href="#top">Top</a></p>
+
+## Allele.proto
+
+Allele represents a single contiguous change
+
+<a name="org.ga4gh.vmc.Allele"/>
+### Allele
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| sequence | [SequenceReference](#org.ga4gh.vmc.SequenceReference) | required | sequence reference (namespace and accession) |
+| location | [Interval](#org.ga4gh.vmc.Interval) | required | location of sequence change |
+| replacement | [string](#string) | required | replacement sequence; empty for deletion |
+
+
+
+
+
+
+<a name="Diplotype.proto"/>
+<p align="right"><a href="#top">Top</a></p>
+
+## Diplotype.proto
+
+Haplotype represents a collection of phased changes on a single
+reference.
+
+<a name="org.ga4gh.vmc.Diplotype"/>
+### Diplotype
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| haplotypes | [Haplotype](#org.ga4gh.vmc.Haplotype) | repeated | phased changes; intervals should not overlap |
+
+
+
+
+
+
+<a name="Genotype.proto"/>
+<p align="right"><a href="#top">Top</a></p>
+
+## Genotype.proto
+
+Genotype represents multiple changes at a single location
+
+<a name="org.ga4gh.vmc.Genotype"/>
+### Genotype
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| sequence | [SequenceReference](#org.ga4gh.vmc.SequenceReference) | required | sequence reference (namespace and accession) |
+| location | [Interval](#org.ga4gh.vmc.Interval) | required | location of sequence change |
+| replacements | [string](#string) | repeated | replacement sequences; empty for deletions |
+
+
+
+
+
+
+<a name="Haplotype.proto"/>
+<p align="right"><a href="#top">Top</a></p>
+
+## Haplotype.proto
+
+Haplotype represents a collection of phased changes on a single
+reference.
+
+<a name="org.ga4gh.vmc.Haplotype"/>
+### Haplotype
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| sequence | [SequenceReference](#org.ga4gh.vmc.SequenceReference) | required | sequence reference (namespace and accession) |
+| interval_edits | [IntervalEdit](#org.ga4gh.vmc.IntervalEdit) | repeated | phased changes; intervals should not overlap |
+
+
+
+
+
+
+<a name="Interval.proto"/>
+<p align="right"><a href="#top">Top</a></p>
+
+## Interval.proto
+
+Represents the definite (not fuzzy) location of a sequence
+feature using interbase coordinates.
```

Comment:
Yes, it's different. Unfortunately, you've missed a lot of conversations that I can't replay here.